File: Basics/For-Loops/Lab/vowels_sum.py
text = input()
vowels_sum = 0

for char in text:
    vowels_value = 0

    if char == "a":
        vowels_value = 1
    elif char == "e":
        vowels_value = 2
    elif char == "i":
        vowels_value = 3
    elif char == "o":
        vowels_value = 4
    elif char == "u":
        vowels_value = 5

    vowels_sum += vowels_value

print(vowels_sum)

File: JavaScript/Basics/0.4-For-Loops/Lab/vowels-sum.js
function vowelsSum(input) {
    let text = input[0];
    let mapper = {
        "a": 1,
        "e": 2,
        "i": 3,
        "o": 4,
        "u": 5 
    }
    let result = 0;
    for (i = 0; i < text.length; i ++) {
        if (text[i].toLowerCase() in mapper) {
            result += mapper[text[i].toLowerCase()]
        }
    }
    console.log(result)
}

vowelsSum(["hello"])
vowelsSum(["hi"])
vowelsSum(["bamboo"])
vowelsSum(["beer"])